web/src/components/apps/DashboardLicenseCard.jsx

```diff
@@ -283,41 +283,42 @@ export default class DashboardLicenseCard extends React.Component {
                     style={{ margin: "10px 0" }}
                   >
                     <span
-                      className={`u-fontWeight--bold`}
+                      className={`u-fontWeight--bold u-cursor--pointer`}
                       style={{ whiteSpace: "pre" }}
-                    >
-                      View {size(appLicense?.entitlements)} license entitlements
-                    </span>
-                    <span
-                      className={`icon clickable ${
-                        this.state.isViewingLicenseEntitlements
-                          ? "up-arrow-icon"
-                          : "down-arrow-icon"
-                      } u-marginLeft--5`}
                       onClick={(e) => {
                         e.stopPropagation();
                         this.viewLicenseEntitlements();
                       }}
-                    />
+                    >
+                      View {size(appLicense?.entitlements)} license entitlements
+                      <span
+                        className={`icon clickable ${
+                          this.state.isViewingLicenseEntitlements
+                            ? "up-arrow-icon"
+                            : "down-arrow-icon"
+                        } u-marginLeft--5`}
+                      />
+                    </span>
                   </span>
                 )}
-                {this.state.isViewingLicenseEntitlements ? (
-                  <LicenseFields
-                    entitlements={appLicense?.entitlements}
-                    entitlementsToShow={this.state.entitlementsToShow}
-                    toggleHideDetails={this.toggleHideDetails}
-                    toggleShowDetails={this.toggleShowDetails}
-                  />
+                {appLicense.entitlements.length > 0 &&
```

Comment:
had to switch up the logic so that if entitlements are less than 5 we render with a div + plus margin top.